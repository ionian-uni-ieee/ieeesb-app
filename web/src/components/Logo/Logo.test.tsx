import React from 'react'

import { render } from '@testing-library/react'

import Logo from './Logo'

describe('Logo', () => {
  it('should render', () => {
    const { container } = render(<Logo />)
    expect(container).toMatchSnapshot()
  })
})
