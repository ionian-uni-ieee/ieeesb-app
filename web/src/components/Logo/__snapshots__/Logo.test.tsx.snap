// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo should render 1`] = `
<div>
  <span
    class="
        logo
        
      "
  >
    <svg
      viewBox="0 0 914 231"
    >
      logo.svg
    </svg>
  </span>
</div>
`;
